example/deploy/crds/mellanox.com_v1alpha1_nicclusterpolicy_cr.yaml

```diff
@@ -31,8 +31,12 @@ spec:
         "configList": [
           {
             "resourceName": "hca_shared_devices_a",
-            "rdmaHcaMax": 1000,
-            "devices": ["ens2f0"]
+            "rdmaHcaMax": 100,
+            "selectors": {
+              "vendors": ["15b3"],
```

Comment:
can you keep just ifNames in the example ?

Reply:
changed